apps/examples/postcss.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  plugins: {
+    'postcss-react-strict-dom': {
+      include: [
+        'src/**/*.{js,jsx,ts,tsx}',
+        '../../packages/react-strict-dom/dist/**/*.{js,jsx,ts,tsx}'
```

Comment:
I used `require.resolve('react-strict-dom')` in #239.
See https://github.com/javascripter/postcss-react-strict-dom/issues/3